src/main/java/io/vertx/codegen/type/AnnotationTypeInfo.java

```diff
@@ -0,0 +1,76 @@
+package io.vertx.codegen.type;
+
+import io.vertx.codegen.Helper;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * @author <a href="mailto:cafeinoman@openaliasbox.org>Francois Delalleau</a>
+ */
+public class AnnotationTypeInfo {
```

Comment:
this should be named differently as this is not an annotation type but rather an annotation value, so something like `AnnotationValueInfo` seems better